api/types/server.go

```diff
@@ -136,6 +138,11 @@ func (s *ServerV2) GetKind() string {
 
 // GetSubKind returns resource sub kind
 func (s *ServerV2) GetSubKind() string {
+	// if the server is a node subkind isn't set, this is a teleport node.
+	if s.Kind == KindNode && s.SubKind == "" {
+		return SubKindTeleportNode
+	}
```

Comment:
nit: this can be simplified:
```suggestion
	if s.SubKind == "" {
		return SubKindTeleportNode
	}
```

Reply:
That would return `SubKindTeleportNode` subkind for Proxy or Auth server resources though